src/components/main-form.js

```diff
@@ -1,180 +1,192 @@
 // References:
 // https://www.bankrate.com/calculators/mortgages/new-house-calculator.aspx
 // https://www.nerdwallet.com/mortgages/mortgage-rates/30-year-fixed
+// https://michaelbluejay.com/house/15vs30.html
 
 import _ from 'lodash';
 import React from 'react';
 import numeral from 'numeral';
-import { InputGroup, InputGroupAddon, InputGroupText, Input } from 'reactstrap';
+import styled, { css } from "react-emotion";
+import { Input } from 'reactstrap';
 import { FV, PMT } from 'formulajs/lib/financial';
 
+
+const COMPOUND_FREQUENCY = 12;
+
+const o = {
```

Comment:
What does the `o` stand for?

Reply:
"option". o1 and o2 statnd for option1 and option2. I can rename if you'd like

Reply:
yeah, as a constant, this can be `DEFAULT_OPTION`

Reply:
no problem. I'm not used to putting objects into constants